src/screens/UserManagement/UserManagementUtils.res

```diff
@@ -81,33 +81,29 @@ let roleType = roleListData =>
 let getArrayOfPermissionData = json => {
   json
   ->LogicUtils.getDictFromJsonObject
-  ->LogicUtils.getArrayFromDict("permissions", [])
+  ->LogicUtils.getArrayFromDict("groups", [])
   ->Array.map(i => i->JSON.Decode.string->Option.getOr(""))
 }
 
 let updatePresentInInfoList = (infoData, permissionsData) => {
   let copyOfInfoData = infoData->Array.copy
   let copyOfPermissionsData = permissionsData->Array.copy
 
-  copyOfInfoData->Array.map((infoValItem: ProviderTypes.getInfoType) => {
-    infoValItem.permissions->Array.forEachWithIndex((
-      enumValue: ProviderTypes.permissions,
-      index,
-    ) => {
-      if copyOfPermissionsData->Array.includes(enumValue.enum_name) {
-        enumValue.isPermissionAllowed = true
-      }
-      infoValItem.permissions[index] = enumValue
-    })
+  copyOfInfoData->Array.map((infoValItem: UserManagementTypes.getInfoType) => {
+    if copyOfPermissionsData->Array.includes(infoValItem.module_) {
+      infoValItem.isPermissionAllowed = true
+    } else {
+      infoValItem.isPermissionAllowed = false
+    }
     infoValItem
   })
 }
 
 let defaultPresentInInfoList = infoData => {
```

Comment:
this function is not required if we remove` src/screens/UserManagement/UserManagementTypes.res` type permission

Reply:
This `isPermissionAllowed` is for module level permissions

Reply:
https://github.com/juspay/hyperswitch-control-center/pull/471#discussion_r1510855022